### ISSUE TITLE
What about ASP.NET Core (Identity Core)

### ISSUE THREAD

**Issue by ikourfaln** [RockstarLabs/oauthforaspnet#20]
Hi,
is those configurations of OAuth providers works for asp.net core too ?

And Thanx for all your hard work 😉

**Comment by jerriep** [comment]
Not at the moment. I will hopefully have ASP.NET Core Guides eventually






On Mon, Apr 18, 2016 at 2:08 PM -0700, "IKOURFALN Slimane" <notifications@github.com> wrote:












Hi,

is those configurations of OAuth providers works for asp.net core too ?



And Thanx for all your hard work 😉

**Comment by ikourfaln** [comment]
Thanx @jerriep for replay.
What about those providers here: [AspNet.Security.OAuth.Providers](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers), are those good for production ?
I see that you are a contributor too of this repo.

Thanx again

**Comment by jerriep** [comment]
Yes, if you believe that ASP.NET Core in its current state is good enough
for you to use in production, then those providers will certainly be good
enough ;)

On Tue, Apr 19, 2016 at 3:36 PM, IKOURFALN Slimane <notifications@github.com

**Comment by ikourfaln** [comment]
Thanx a lot @jerriep for fast replay.
have a nice day
